src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/ViewDataDictionaryControllerPropertyActivator.cs

```diff
@@ -14,24 +13,30 @@ namespace Microsoft.AspNetCore.Mvc.ViewFeatures
     public class ViewDataDictionaryControllerPropertyActivator : IControllerPropertyActivator
```

Comment:
IMO, this should have been an internal type. It does not have a whole lot of use being in a public namespace.